test/client/fixtures/sandbox/event/event-simulator-test.js

```diff
@@ -210,7 +210,8 @@ test('event with coords (clientX, clientY, screenX, screenY)', function () {
     var screenY = 4;
 
     domElement.onmousedown = function (e) {
-        var coordsAreCorrect = e.clientX === clientX && e.clientY === clientY && e.screenX === screenX && e.screenY === screenY;
+        var coordsAreCorrect = e.clientX === clientX && e.clientY === clientY && e.screenX === screenX && e.screenY ===
```

Comment:
It doesn't really matter, but it looks better if we write
```
var foo = bar1 === bar1 &&
          bar1 === bar2;
```

Reply:
looks like it was autoformatted accidentally